internal/handles.go

```diff
@@ -200,8 +200,7 @@ func (parent *Inode) removeChildUnlocked(inode *Inode) {
 	}
 	i := sort.Search(l, parent.findInodeFunc(*inode.Name, inode.isDir()))
 	if i >= l || *parent.dir.Children[i].Name != *inode.Name {
-		panic(fmt.Sprintf("%v.removeName(%v) but child not found: %v",
```

Comment:
probably shouldn't be part of this?

Reply:
It came from https://github.com/kahing/goofys/pull/292. I'll remove it.